lib/ohai/plugins/darwin/network.rb

```diff
@@ -50,6 +50,14 @@ def parse_media(media_string)
     media
   end
 
+  def darwin_parse_netmask(netmask)
```

Comment:
Can you toss a YARD comment on these explaining what the arg and return is for anyone that needs to touch it in the future. Also thanks for keeping the names unique so we avoid collisions in the future.

Reply:
@tas50  I"m pretty sure there these are the same, and thus should not have different names and should not be in different places, no?